tests/airport.spec.js

```diff
@@ -63,5 +63,32 @@ plane = null;
 
 console.log(`---------------------------------------------`);
 
+// // TEST 6: 
+console.log(`---------------------------------------------`)
+console.log(`Test 6: atDefaultCapacity() returns true if airportPlanes has reached it's defaultCapacity`);
+
+//Arrange
+airport = new Airport();
+plane = new Plane();
+expected = true;
```

Comment:
Should have used assertTrue from the framework from the beginning. I changed it later.